content/spin/api-guides-overview.md

```diff
@@ -18,6 +18,6 @@ The following table shows the status of the interfaces Spin provides to applicat
 | [PostgreSQL](/spin/rdbms-storage)                             | Experimental | Yes |
 | [MySQL](/spin/rdbms-storage)                                  | Experimental | Yes |
 | [Key-value Storage](/spin/kv-store-api-guide)                      | Stabilizing | Yes |
-| [SQLite Storage](/spin/sqlite-api-guide)                      | Experimental | No |
+| [SQLite Storage](/spin/sqlite-api-guide)                      | Experimental | [Private Beta](/cloud/cloud/noops-sql-db.md) |
```

Comment:
This seems to be have `cloud` twice in the path.  Usually it would appear only once.  Does that link actually work?

Reply:
Looks like not 

![image](https://github.com/fermyon/developer/assets/865538/5e1dcb95-525b-436d-ac1c-e81a2be2cd07)

```suggestion
| [SQLite Storage](/spin/sqlite-api-guide)                      | Experimental | [Private Beta](/cloud/noops-sql-db.md) |
```